lib/ex_wallet/ripple_address.ex

```diff
@@ -0,0 +1,38 @@
+defmodule ExWallet.RippleAddress do
+  @moduledoc """
+  Generates a Ripple network's compatible address account. Ripple address
+  starts with `r`.
+  """
+  alias ExWallet.{Compression, KeyPair}
+
+  @doc """
+  Generates an address from given public key. The public key can be compressed
+  with 33 bytes or their original size with 65.
+  """
+  def from_public_key(pub_key) when byte_size(pub_key) == 33,
```

Comment:
Instead of doc + magic number would it make sense to use compiled-variable?

Reply:
in this case not compiled variable, but a guard clause

doc + guard clause 👍